### PR TITLE
First Commit in Student

### PR DESCRIPTION
**Pull request by Bharat-Fexle-May2022**

### CODE REVIEW

File: src/classes/StudentQ.cls [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            
+        //Fees Details
+        student.Current_Year__c = '1st Year';
+        student.Is_Donation__c = False;
+        student.Donation_Amount__c = 0;
+        student.Fees_Submitted__c = True;
+        student.Pending_Fees_Amount__c = 50;
+        student.Exam_Fees_Submitted__c = True;
+        student.Pending_Exam_Fees__c = 0;
+        //student.Pending_Overall_Amount__c = ; //Not Writable
```
Comment (SeemuSaikiaSF) [inline src/classes/StudentQ.cls:37]: Make it Writable

File: src/classes/StudentQ.cls [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        student.Father_Mobile_Number__c = '9184873834';
+        student.Mobile_Number__c = '9638491657';
+        student.Email__c = 'abhay.kevat@gmail.com';
+        student.Home_Phone__c = '9184873834';
+        student.Overall_Grade__c = 'A';
+        student.Address__c = 'Vrindavan';
+        student.City__c = 'Mathura';
+        student.Pin_Code__c = 281003;
+        student.State__c = 'Uttar Pradesh';
+        student.RecordTypeId = '0125i000000LVYbAAO';
```
[the remark this answers is not in the document]
Comment (SeemuSaikiaSF) [reply src/classes/StudentQ.cls:22]: Create Students records with both record types.

File: src/classes/StudentQ.cls [hunk 3]
```diff
@@ -0,0 +1,66 @@
+public class StudentQ {
+    public StudentQ() {
+        Student__c student = new Student__c();
+        
+        //Registrar
+        
+        //Personal Information
+        Date myDate = System.today();
+        student.Date_Of_Birth__c = myDate;
```
Comment (SeemuSaikiaSF) [inline src/classes/StudentQ.cls:9]: Add valid date , use past years date

File: src/classes/StudentQ.cls (hunk 1 above)
Comment (Bharat-Fexle-May2022) [reply src/classes/StudentQ.cls:37]: Sir i use the Formula Data Type for Pending Overall Amount = Pending Fees Amount + Pending Exam Fees

**Pull request closed without merging**